sw/device/lib/crypto/drivers/kmac_test.c

```diff
@@ -0,0 +1,45 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/crypto/drivers/kmac.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+extern size_t nist_kmac_nr_of_vectors;
+extern kmac_test_vector_t* nist_kmac_vectors[];
+
+OTTF_DEFINE_TEST_CONFIG();
+bool test_main(void) {
+  
+  kmac_error_t err;
+  //kmac_csr_context_t csr_context;
```

Comment:
nit: I think this commented-out code can be removed.

Reply:
It shouldn't be here, I will remove it.